### ISSUE TITLE
Make scRNAseq queries live (EPIC)

### ISSUE THREAD

**Issue by dosumis** [VirtualFlyBrain/VFB2#345]
Pushing scRNAseq data queries live: Can we do it this week?

* Done (but double check status)
    * Data already live in DB	
    * Working queries in place
* Add query hooks to neurons (driven by scRNAseq flag) - already done?
    * Hack in via Cypher.  Don't wait for another DB build.
* TODO
   * Code needed to translate queries to results tables
   *  Can we make sure code edits are not held up by clashing updates for other purposes?

**Comment by Robbie1977** [comment]
```MATCH (primary:Class:Anatomy)<-[r:SUBCLASSOF*0..]-(a:Class:Anatomy)<-[r1:composed_primarily_of]-(c:Cluster:Individual)-[r3:has_source]->(ds:scRNAseq_DataSet:Individual) RETURN * LIMIT 1```

**Comment by dosumis** [comment]
@hkir-dev  to work on moving anatomy -> cluster query to SOLR.

**Comment by dosumis** [comment]
Items for follow-on sprint(s):

-  Update to latest build 
    - Fix queries to cope with schema change
    -  Fix xref-roller so that it generates multiple xrefs when accession is list with n > 1.
